snuba/admin/static/welcome/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+function Welcome() {
+  const PROD_URL_START = "https://snuba-admin";
+
+  const AVAILABLE_REGIONS =
+    "https://www.notion.so/sentry/Snuba-Admin-8344d5d508014d4a867cdb1b8e8d22cf";
```

Comment:
Maybe we should move this to settings. Feels weird having this in a public repo.